fhd_core/beam_modeling/beam_setup.pro

```diff
@@ -156,7 +159,8 @@ FOR pol_i=0,n_pol-1 DO BEGIN
             t_beam_power+=Systime(1)-t_bpwr
             t_bint=Systime(1)
             ;divide by psf_resolution^2 since the FFT is done at a different resolution and requires a different normalization
-            beam_int+=baseline_group_n*Total(Abs(psf_base_superres)^2,/double)/psf_resolution^2. 
+            beam_int+=baseline_group_n*Total(psf_base_superres,/double)/psf_resolution^2. 
```

Comment:
Should the beam integral be over the absolute value of the beam?

Reply:
In this case, beam_int leads to the primary_beam_area, and beam2_int gives primary_beam_sq_area. The latter is what I'm proposing we rename beam_integral in obs.

primary_beam_area as defined here gives the correct quantity for power spectrum normalization with the PAPER pipeline, so I need to have it in the obs.